blog/2015/12/workshop-in-stockholm.html

```diff
@@ -0,0 +1,180 @@
+---
+layout: blog
+root: ../../..
+author: "Roman Valls"
```

Comment:
My full name is "Roman Valls Guimera"... spaniards with multiple surnames ;)

Otherwise it all looks great, thanks Greg!